controllers/v1/users.js

```diff
@@ -142,18 +142,21 @@ module.exports = class Users {
 	 * @param {String} req.pageSize - pageSize
 	 * @param {String} req.searchText - searchText
 	 * @param {String} req.query.isAPrivateProgram - isAPrivateProgram
+	 * @param {String} req.query.getProjectsCount - getProjectsCount under the program
 	 * @returns {Object} list of targeted user programs.
 	 */
 
 	programs(req) {
 		return new Promise(async (resolve, reject) => {
 			try {
 				let isAPrivateProgram = UTILS.convertStringToBoolean(req.query.isAPrivateProgram)
+				req.query.getProjectsCount = UTILS.convertStringToBoolean(req.query.getProjectsCount)
 
 				if (isAPrivateProgram) {
 					let programsData = await usersHelper.privatePrograms(
 						req.userDetails.userInformation.userId,
-						req.query.language ? req.query.language : ''
+						req.query.language ? req.query.language : '',
```

Comment:
@praveenKDass Please introduce pagination to this API

Reply:
@VISHNUDAS-tunerlabs  added